pkg/util/k8s/k8s.go

```diff
@@ -730,3 +730,40 @@ func CreateOrUpdateStorageNodeStatus(
 	}
 	return nil
 }
+
+// GetDaemonSetPods returns a list of pods for the given daemon set
+func GetDaemonSetPods(
+	k8sClient client.Client,
+	ds *appsv1.DaemonSet,
+) ([]*v1.Pod, error) {
+	return GetPodsByOwner(k8sClient, ds.UID, ds.Namespace)
+}
+
+// GetPodsByOwner returns pods for a given owner and namespace
+func GetPodsByOwner(
```

Comment:
Can we not use https://github.com/portworx/sched-ops/blob/master/k8s/k8s.go#L2564?

Reply:
This gets it from the cache. We are using the client from sdk everywhere to reduce api server calls as much as possible.